src/shared/layouts/BaseLayout/components/TopBar/Breadcrumbs/Breadcrumbs.hooks.ts

```diff
@@ -131,13 +134,15 @@ export const useBreadcrumbs = (restCrumbs?: Breadcrumb[]) => {
           : page.dashboardRespondents,
       });
     }
+
     if (respondentId) {
       newBreadcrumbs.push({
-        icon: 'account',
+        icon: isMultiInformant ? undefined : 'account',
         label: respondentLabel || subjectLabel,
         disabledLink: true,
       });
     }
+
```

Comment:
I'm not sure which part of breadcrumbs code is responsible for this, but I just noted a couple issues:

- The breadcrumbs don't seem to always populate based on your current route. If you tap the Schedule tab, the "Schedule" breadcrumb gets added (which seems correct), but if you tap the "Connections" tab, you _don't_ get a "Connections" breadcrumb – but it seems logical to me that you should.
- The respondent's ID breadcrumb isn't clickable – not sure if that's part of the requirements, but I would expect that if I'm on a respondent details subscreen, I should be able to click the respondent ID and it would take me to the default URL (`/dashboard/[applet-id]/respondents/[respondent-id]`). However, may be a question for product/design?

https://github.com/ChildMindInstitute/mindlogger-admin/assets/1282772/6113c6dc-3a83-4fe3-8ca9-34d3e1498b7e

Reply:
Looking at the designs, the correct behavior seems to be that we don't get either a "Connections" breadcrumb _or_ a "Schedule" breadcrumb: https://www.figma.com/file/6BNx63peNVzrHAF1FoZ78x/Multi-Informant-2024?type=design&node-id=895-109325&mode=design&t=9Q84A50oR1kTbL3t-4 and https://www.figma.com/file/6BNx63peNVzrHAF1FoZ78x/Multi-Informant-2024?type=design&node-id=895-109690&mode=design&t=9Q84A50oR1kTbL3t-4. We'll want the removal behind that `isMultiInformant` check that you have